src/components/multi-action-button/multi-action-button.stories.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, select, boolean } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import MultiActionButton from './multi-action-button';
+import notes from './notes.md';
+import Button from '../button/button';
+import OptionsHelper from '../../utils/helpers/options-helper';
+
+storiesOf('Multi Action Button', module)
+  .addParameters({
+    info: {
+      propTablesExclude: [Button]
+    }
+  })
+  .add('default', () => {
+    const disabled = boolean('disabled', false);
+    const label = text('text', 'Example Multi Action Button');
+    const as = label ? select('as', ['primary', 'secondary', 'transparent'], 'secondary') : undefined;
+    const align = select('align', OptionsHelper.alignBinary, OptionsHelper.alignBinary[0]);
+
+    return (
```

Comment:
This shouldn't cover the component description when open.

![image](https://user-images.githubusercontent.com/26925206/53879115-8dd3e880-4005-11e9-981b-57a19594480e.png)

Reply:
Fixed 👍